core/blockly.js

```diff
@@ -141,36 +141,30 @@ Blockly.svgSize = function(svg) {
 };
 
 /**
- * Schedule a call to the resize handler.  Groups of simultaneous events (e.g.
- * a tree of blocks being deleted) are merged into one call.
- * @param {Blockly.WorkspaceSvg} workspace Any workspace in the SVG.
+ * Size the workspace when the contents change.  This also updates
+ * scrollbars accordingly.
+ * @param {!Blockly.WorkspaceSvg} opt_workspace. The svg group of the workspace
+ *   to resize. If the workspace isn't specified, the main workspace will be
+ *   used.
  */
-Blockly.asyncSvgResize = function(workspace) {
-  if (Blockly.svgResizePending_) {
-    return;
-  }
+Blockly.resizeSvgContents = function(opt_workspace) {
+  var workspace = opt_workspace;
   if (!workspace) {
```

Comment:
In what case would workspace be false?  All callers of this function seem to provide a workspace.  The getMainWorkspace function should be treated with caution since it opens potential race conditions if there are multiple workspaces on a page (e.g. Block Factory).